src/containers/new-dao.js

```diff
@@ -68,6 +68,12 @@ const mapDispatchToProps = (dispatch, ownProps) => {
       //TODO: Call SemadaCore.checkProposal() to close validation pool
       
     },
+    saveSemBalance: async() => {
```

Comment:
Same change here as join dao- shouldn't be calling semada core to update sem balance. Just refresh on a timer from app wrapper.